src/sentry/receivers/rule_snooze.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models.signals import post_save, pre_delete
+
+from sentry.models.rulesnooze import RuleSnooze
+from sentry.workflow_engine.models import AlertRuleDetector, AlertRuleWorkflow
+
+
+def _update_workflow_engine_models(instance: RuleSnooze, is_enabled: bool) -> None:
+    if instance.user_id is not None:
+        return
```

Comment:
What does having a user id indicate here?

Reply:
that the rule / alert rule is only snoozed for a specific user